Reference/Templating/index.md

```diff
@@ -13,7 +13,7 @@ By default Umbraco uses MVC templates.
 
 #### [Working with MVC (views, razor, etc...)](Mvc/index.md)
 
-Describes how to work with MVC views, the razor syntax and APIs available as well details on how to create forms, step-by-step guides and other advanced techniques.
+Describes how to work with MVC views, the razor syntax, and APIs available, as well details on how to create forms, step-by-step guides and other advanced techniques.
```

Comment:
While we're adding some commas in here, how about we tweak the text too?

'as well details on how to create forms' 

could become

'as well as details on how to create forms'

etc